src/private/MDMPerformerGroup.m

```diff
@@ -72,6 +72,19 @@ - (void)executeLog:(MDMTransactionLog *)log trace:(MDMTrace *)trace {
     if ([performer respondsToSelector:@selector(addPlan:)]) {
       [(id<MDMPlanPerforming>)performer addPlan:plan];
     }
+    
+    // MDMNamedPlanPerforming callbacks
+    if (log.name.length) {
+      if (log.isRemoval) {
+        if ([performer respondsToSelector:@selector(removePlanNamed:)]) {
+          [(id<MDMNamedPlanPerforming>)performer removePlanNamed:log.name];
+        }
+      } else {
+        if ([performer respondsToSelector:@selector(addPlan:named:)]) {
+          [(id<MDMNamedPlanPerforming>)performer addPlan:plan named:log.name];
+        }
+      }
```

Comment:
I don't strictly have a `performerForName` method in `executeLog` method in `MDMPerformerGroup`, but the existing `performerForPlan:isNew:` method seems to do the trick.
Would there be any scenario whereby a performed associated with a named plan would need to be looked up by name instead of by it's plan's performer?